definitions/redux_v3.x.x/flow_>=v0.23.x/redux_v3.x.x.js

```diff
@@ -0,0 +1,29 @@
+declare module 'redux' {
+  declare type State = any;
+  declare type Action = Object;
```

Comment:
Redux only enforces that it isn’t `undefined`. Some people really want to use Symbols 😄 .